code/go/internal/validator/semantic/version_integrity.go

```diff
@@ -40,6 +43,20 @@ func ValidateVersionIntegrity(fsys fspath.FS) ve.ValidationErrors {
 	return nil
 }
 
+// ValidateTopChangelogLink returns validation errors if the link is not a valid PR github.com link.
+// If the link is not a github.com link this validation is skipped and does not return an error.
+func ValidateTopChangelogLink(fsys fspath.FS) ve.ValidationErrors {
```

Comment:
Is there any reason why you validate only the top link? I think it's feasible to go over all entries.

Reply:
I wasn't sure if it was okay to validate all the older links as that could break builds for something that was in built a while ago and other changes were built on top of it.

Reply:
The code now checks all links.